ckanext/qgov/common/test_mime_type_validation.py

```diff
@@ -0,0 +1,57 @@
+# encoding: utf-8
+
+'''Tests for the ckanext.qgov extension MIME type validation.
```

Comment:
can't this new functionality be its own tiny plugin?

Reply:
I'm actually hoping to integrate a variation on this into core at some point (maybe just the comparison and canonicalisation of MIME type, not the whitelist).